### PR TITLE
Fixes #7 - Rewrite subaccount theme script with playwright and CanvasAPI

### LINKED ISSUE #7: Use Playwright for Canvas subaccount customized theme report
The theme report script currently use Selenium. We will upgrade and use the more widely supported Playwright library.

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

File: theme_report/README.md [hunk 1]
```diff
@@ -1,23 +1,13 @@
 ## Installation instructions
 
-- install [Playwright for Python](https://playwright.dev/python/docs/intro):
-  Install the Pytest plugin:
-  `pip install pytest-playwright`
+- install required libraries
```
Comment (jonespm) [inline theme_report/README.md:3]: Should still include more comments about what the values in the env.json mean.

File: theme_report/generate_theme_report.py [hunk 2]
```diff
@@ -1,118 +1,176 @@
...
+        if session_url is not None:
+            self.page=self.page.context.new_page()
+            self.page.goto(session_url)
+            logging.info("Session URL was not none and was hit.")
+
+    # Run the script
+    # Will launch a browser and navigate to the subaccount
+    def run(self, playwright: Playwright) -> None:
+
+        browser = playwright.chromium.launch(headless=False)
```
Comment (jonespm) [inline theme_report/generate_theme_report.py]: Should still make headless configurable

**Review by jaydonkrooss — APPROVED** [review]

File: theme_report/env_sample.json [hunk 3]
```diff
@@ -1,5 +1,7 @@
 {
 	"API_URL": "",
 	"API_KEY": "",
-	"CANVAS_URL": "https://instructure.com/accounts/"
+	"ROOT_ACCOUNT": "1",
+	"DISABLE_RESOURCES": "true",
+	"HEADLESS": "false"
```
Comment (zqian) [inline theme_report/env_sample.json]: According to the README file, this parameter should be 

"HEADLESS_BROWSER": false

NOTE the param is of boolean type, not String.

File: theme_report/README.md [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
 - execute the script to generate theme_report.csv file:
-  `python ./generate_theme_report.py`
+  `python3 ./generate_theme_report.py`
+
+## Variables used in env.json
+
+* `API_URL` (str): Full API URL of the instance where API_KEY was generated starting with https://
+* `API_KEY` (str): API Key generated from Canvas settings for an admin account
+* `ROOT_ACCOUNT` (int): Root account number
+* `DISABLE_RESOURCES` (bool): Set or disable resources retrieval to improve speed
```
Comment (zqian) [inline theme_report/README.md]: maybe add default setting values here for both 'DISABLE_RESOURCES' and 'HEADLESS_BROWSER'

File: theme_report/env_sample.json [hunk 5]
```diff
@@ -1,5 +1,7 @@
 {
 	"API_URL": "",
 	"API_KEY": "",
-	"CANVAS_URL": "https://instructure.com/accounts/"
+	"ROOT_ACCOUNT": "1",
+	"DISABLE_RESOURCES": "true",
```
Comment (zqian) [inline theme_report/env_sample.json]: should use boolean value instead of string value for "DISABLE_RESOURCES"

**Review by zqian — CHANGES_REQUESTED** [review]
Added comments for the env settings. In general, the new version of script runs much faster than the prior version.

File: theme_report/env_sample.json (hunk 3 above)
Comment (jonespm) [reply theme_report/env_sample.json]: Okay I'll update these, I left them unset so they just used the defaults when I was testing.

File: theme_report/README.md (hunk 4 above)
Comment (jonespm) [reply theme_report/README.md]: Okay will add it.

**Review by zqian — APPROVED** [review]
The script ran successfully.

**Pull request merged**